Protos/V1/filters_common.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package armonik.api.grpc.v1;
+
+import "google/protobuf/timestamp.proto";
+import "result_status.proto";
+import "session_status.proto";
+import "task_status.proto";
+
+option csharp_namespace = "Armonik.Api.gRPC.V1";
+
+enum FilterStringOperator {
+  FILTER_STRING_OPERATOR_UNSPECIFIED = 0; /** Unspecified */
+  FILTER_STRING_OPERATOR_EQUAL = 1; /** Equal */
+  FILTER_STRING_OPERATOR_NOT_EQUAL = 2; /** Not equal */
+  FILTER_STRING_OPERATOR_CONTAINS = 3; /** Contains */
+  FILTER_STRING_OPERATOR_NOT_CONTAINS = 4; /** Not contains */
+  FILTER_STRING_OPERATOR_STARTS_WITH = 5; /** Starts with */
+  FILTER_STRING_OPERATOR_ENDS_WITH = 6; /** Ends with */
+}
+
+message FilterString {
+  string value = 1;
+  FilterStringOperator operator = 2;
+}
+
+enum FilterNumberOperator {
+  FILTER_NUMBER_OPERATOR_UNSPECIFIED = 0; /** Unspecified */
+  FILTER_NUMBER_OPERATOR_LESS_THAN = 1; /** Less than */
+  FILTER_NUMBER_OPERATOR_LESS_THAN_OR_EQUAL = 2; /** Less than or equal */
+  FILTER_NUMBER_OPERATOR_EQUAL = 3; /** Equal */
+  FILTER_NUMBER_OPERATOR_NOT_EQUAL = 4; /** Not equal */
+  FILTER_NUMBER_OPERATOR_GREATER_THAN_OR_EQUAL = 5; /** Greater than or equal */
+  FILTER_NUMBER_OPERATOR_GREATER_THAN = 6; /** Greater than */
+}
+
+message FilterNumber {
+  int64 value = 1;
+  FilterNumberOperator operator = 2;
+}
+
+enum FilterDateOperator {
+  FILTER_DATE_OPERATOR_UNSPECIFIED = 0; /** Unspecified */
+  FILTER_DATE_OPERATOR_BEFORE = 1; /** Before */
+  FILTER_DATE_OPERATOR_BEFORE_OR_EQUAL = 2; /** Before or equal */
+  FILTER_DATE_OPERATOR_EQUAL = 3; /** Equal */
+  FILTER_DATE_OPERATOR_AFTER_OR_EQUAL = 4; /** After or equal */
+  FILTER_DATE_OPERATOR_AFTER = 5; /** After */
+  FILTER_DATE_OPERATOR_NOT_EQUAL = 6; /** Not equal */
```

Comment:
`NOT_EQUAL` should be grouped with `EQUAL`.

Reply:
yes, will update this.